aries-site/src/layouts/content/Example.js

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import { Box, Button, Drop, Text, ThemeContext } from 'grommet';
+import { Code, Document, Template } from 'grommet-icons';
+import Prism from 'prismjs';
+
+import { colors } from '../../themes/aries';
+
+const syntax = {
+  dark: styled.pre`
+    background: transparent;
+    .string { color: #6c9; }
+    .keyword { color: #c9c; }
+    .function { color: #f93; }
+    .operator { color: #6cc; }
+    .tag { color: #f66; }
+    .script { color: #ccc; }
+    .punctuation { color: #ccc; }
+    .attr-name { color: #6c9; }
+    .attr-value { color: #c9c; }
+  `,
+  light: styled.pre`
+    background: transparent;
+    .string { color: #690; }
+    .keyword { color: #069; }
+    .function { color: #c66; }
+    .operator { color: #963; }
+    .tag { color: #906; }
+    .script { color: #333; }
+    .punctuation { color: #999; }
+    .attr-name { color: #690; }
+    .attr-value { color: #069; }
+  `,
+};
+
+const IconButton = ({ title, ...rest }) => {
+  const ref = React.useRef();
+  const [hover, setHover] = React.useState();
+
+  React.useEffect(() => {
+    if (!hover) return undefined;
+    const timer = setTimeout(() => setHover(false), 2000);
+    return () => clearTimeout(timer);
+  }, [hover]);
+
+  return (
+    <>
+      <Button
+        ref={ref}
+        hoverIndicator
+        onMouseEnter={() => setHover(true)}
+        onMouseLeave={() => setHover(false)}
+        title={hover ? undefined : title}
+        {...rest}
+      />
+      {hover && (
+        <Drop
+          target={ref.current}
+          align={{ bottom: 'top', right: 'right' }}
+          plain
+          stretch={false}
+        >
+          <Box
+            background={colors['background-contrast']}
```

Comment:
I think you should be able to use:
```suggestion
            background='background-contrast'
```